schema.sql

```diff
@@ -10,4 +10,78 @@
 
 -- for the second day's project
 
- ALTER TABLE animals ADD COLUMN species VARCHAR(50);
\ No newline at end of file
+ ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+--  day 3 project
+
+CREATE TABLE owners(
+    id SERIAL PRIMARY KEY,
+    full_name CHAR(50),
+    age INT
+);
+
+CREATE TABLE species(
+    id SERIAL PRIMARY KEY,
+    name CHAR(50)
+);
+
+CREATE SEQUENCE my_id AS integer START 1 OWNED BY animals.id;
+
+ALTER TABLE animals ALTER COLUMN id SET DEFAULT nextval('my_id');
+
+\d animals;
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN
+species_id INTEGER REFERENCES species(id);
+
+SELECT * FROM animals;
+
+ ALTER TABLE animals ADD COLUMN
+  owner_id INTEGER REFERENCES owners(id);
+
+INSERT INTO owners(full_name, age)
+VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+  ('Pokemon'),
+  ('Digimon');
```

Comment:
- Kindly, Everything that populates tables with data should be saved in the data.sql file as you are inserting data here so move it to data.sql to put the correct SQL commands in correct place👍